app/src/androidTest/java/com/android/partagix/viewAccount/ViewAccountTest.kt

```diff
@@ -0,0 +1,376 @@
+          navigationActions = mockNavActions) // mockNavActions::navigateTo
+    }
+
+    onComposeScreen<ViewAccount>(composeTestRule) { username { assertIsDisplayed() } }
+  }
+
+  // usernameText
+  @Test
+  fun usernameWorks() = run {
+    every { mockUserViewModel.uiState } returns emptyMockUiState
+    composeTestRule.setContent {
+      ViewAccount(
+          modifier = Modifier,
+          userViewModel = mockUserViewModel,
+          navigationActions = mockNavActions) // mockNavActions::navigateTo
+    }
+
+    onComposeScreen<ViewAccount>(composeTestRule) {
+      username { assertIsDisplayed() }
+      //            usernameText { assertIsDisplayed() }
+      val username = mockUserViewModel.uiState.value.user.name
+      usernameText { assertTextEquals("$username's profile") }
+    }
+  }
+
+  //    address
+  @Test
+  fun addressIsDisplayed() = run {
+    every { mockUserViewModel.uiState } returns emptyMockUiState
+    composeTestRule.setContent {
+      ViewAccount(
+          modifier = Modifier,
+          userViewModel = mockUserViewModel,
+          navigationActions = mockNavActions) // mockNavActions::navigateTo
+    }
+
+    onComposeScreen<ViewAccount>(composeTestRule) { address { assertIsDisplayed() } }
+  }
+
+  @Test
+  fun addressWorks() = run {
+    every { mockUserViewModel.uiState } returns emptyMockUiState
+    composeTestRule.setContent {
+      ViewAccount(
+          modifier = Modifier,
+          userViewModel = mockUserViewModel,
+          navigationActions = mockNavActions) // mockNavActions::navigateTo
+    }
+
+    onComposeScreen<ViewAccount>(composeTestRule) {
+      address { assertIsDisplayed() }
+      val userAddress = mockUserViewModel.uiState.value.user.address
+      //        address { assertValueEquals(userAddress)}
+      //        addressText { assertIsDisplayed() }
+      //        addressIcon { assertIsDisplayed() }
+    }
+  }
+
+  //    rating
+  @Test
+  fun ratingIsDisplayed() = run {
+    every { mockUserViewModel.uiState } returns emptyMockUiState
+    composeTestRule.setContent {
+      ViewAccount(
+          modifier = Modifier,
+          userViewModel = mockUserViewModel,
+          navigationActions = mockNavActions) // mockNavActions::navigateTo
+    }
+
+    onComposeScreen<ViewAccount>(composeTestRule) { rating { assertIsDisplayed() } }
+  }
+
+  @Test
+  fun ratingWorks() = run {
+    every { mockUserViewModel.uiState } returns emptyMockUiState
+    composeTestRule.setContent {
+      ViewAccount(
+          modifier = Modifier,
+          userViewModel = mockUserViewModel,
+          navigationActions = mockNavActions) // mockNavActions::navigateTo
+    }
+
+    onComposeScreen<ViewAccount>(composeTestRule) {
+      rating { assertIsDisplayed() }
+      //        ratingText { assertIsDisplayed() }
+
+      val rank = mockUserViewModel.uiState.value.user.rank
+      if (rank == "") {
+        //            ratingText { assertTextEquals("No trust yet") }
```

Comment:
Same here, you can add a TODO to restore the tests when the default user is updated.